docs/default_csi_plugin.md

```diff
@@ -31,20 +31,22 @@ Full documentation of the config spec can be found at [github.com/red-hat-storag
 
 #### Path
 
-The user provided lvmd config should be written to the same directory as the MicroShift config.  If a MicroShift config
-doesn't exist, MicroShift will assume default lvmd values. These paths will be checked for the config, depending on the user MicroShift
-is run as.
-
-1. User config dir: `~/.microshift/lvmd.yaml`
-2. Global config dir: `/etc/microshift/lvmd.yaml`
+The user provided lvmd config should be written to the same directory as the MicroShift config.  If an lvmd configuration file
+does not exist in `/etc/microshift/lvmd.yaml`, MicroShift will use default values. 
 
 ## System Requirements
 
-### Volume Group Name
+### Default Volume Group
+
+If there is only one volume group on the system, LVMS uses it by
+default. If there are multiple volume groups, and no configuration
+file, LVMS looks for a volume group named `rhel`. If there is no
+volume group named `rhel`, LVMS is disabled.
 
-The default integration of LVMS assumes a volume-group named `rhel`. LVMS's node-controller expects that volume
-group to exist prior to launching the service. If the volume group does not exist, the node-controller will fail to
-start and enter a CrashLoopBackoff state.
+LVMS expects all volume groups to exist prior to launching the
+service. If LVMS is configured to use a volume group that does not
+exist, the node-controller Pod will fail and enter a CrashLoopBackoff
+state.
```

Comment:
```suggestion
state.


> Note: The VolumeGroup selected by MicroShift may be completely allocated to system and user LogicalVolumes. Be sure the VolumeGroup has sufficient unallocated capacity for workloads and extend the VolumeGroup as necessary.
```

Reply:
Not sure if it's strictly necessary to remind users of this, but it would hopefully head off the inevitable "no space available" bug report.